grpc-auth-support.md

```diff
@@ -26,7 +26,7 @@ server.Serve(lis)
 ## Google Compute Engine (GCE)
 
 ```Go
-conn, err := grpc.Dial(serverAddr, grpc.WithTransportCredentials(credentials.NewClientTLSFromCert(nil, ""), grpc.WithPerRPCCredentials(credentials.NewComputeEngine())))
+conn, err := grpc.Dial(serverAddr, grpc.WithTransportCredentials(credentials.NewClientTLSFromCert(nil, ""), grpc.WithPerRPCCredentials(credentials.TokenSource{google.ComputeTokenSource("")})))
```

Comment:
I actually like NewComputeEngine (and make TokenSource unexported) better because it is consistent with other credentials (service account and default) and simpler.

Plus, this change should not be in this PR.